vendor/bat-native-ads/src/bat/ads/internal/server/server_host_util.h

```diff
@@ -1,21 +1,22 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_SERVER_CONFIRMATIONS_SERVER_UTIL_H_
-#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_SERVER_CONFIRMATIONS_SERVER_UTIL_H_
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_SERVER_SERVER_HOST_UTIL_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_SERVER_SERVER_HOST_UTIL_H_
 
 #include <string>
 
 namespace ads {
-namespace confirmations {
 namespace server {
 
-std::string GetHost();
+std::string GetStaticHost();
```

Comment:
I'd suggest adding a comment above each of these to explain what they're for:

1. `GetStaticHost()`: Requests that are not user-specific and do not process personal data.
2. `GetGeoHost()`: Only used by the `/v1/getstate` endpoint.
3. `GetNonAnonymousHost()`: Requests that include the wallet ID and therefore fully identify the user.
4. `GetAnonymousHost()`: Requests that must not include the wallet ID or allow Brave to link the request to a wallet in any other way.

Reply:
Added